server/dns-service/dnsservice-api/src/main/java/org/apache/james/dnsservice/api/InMemoryDNSService.java

```diff
@@ -17,19 +17,17 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.mpt.smtp.dns;
+package org.apache.james.dnsservice.api;
```

Comment:
I put it here because I have no idea where to put it, where do you think I should put it ?